lib/cosmos/script/script.rb

```diff
@@ -40,43 +77,35 @@ module Script
 
     # Called when this module is mixed in using "include Cosmos::Script"
     def self.included(base)
-      $cmd_tlm_disconnect = false
+      $disconnected_targets = nil
       $cmd_tlm_replay_mode = false
       $cmd_tlm_server = nil
       initialize_script_module()
     end
 
     def initialize_script_module(config_file = CmdTlmServer::DEFAULT_CONFIG_FILE)
-      if $cmd_tlm_disconnect
-        # Start up a standalone CTS in disconnected mode
-        $cmd_tlm_server = CmdTlmServer.new(config_file, false, true)
-      else
-        # Start a Json connect to the real server
-        if $cmd_tlm_replay_mode
-          $cmd_tlm_server = JsonDRbObject.new(System.connect_hosts['REPLAY_API'], System.ports['REPLAY_API'])
-        else
-          $cmd_tlm_server = JsonDRbObject.new(System.connect_hosts['CTS_API'], System.ports['CTS_API'])
-        end
-      end
+      $cmd_tlm_server = ServerProxy.new(config_file)
```

Comment:
Seems like there is some cleanup missing here before creating a new ServerProxy.  Maybe need a ServerProxy.shutdown or something on the old one to close down the @cmd_tlm_server and @disconnected.